tests/operations.go

```diff
@@ -230,6 +229,18 @@ func (ops *Operations) withDefaults() {
 	}
 }
 
+// CheckPodStatusEventually gives the phase of the pod eventually
+func (ops *Operations) CheckPodStatusEventually(pod *corev1.Pod, expectedPodPhase corev1.PodPhase) bool {
+	//var podStatus corev1.PodPhase
+	for i := 0; i < maxRetry; i++ {
```

Comment:
you need to fetch the latest pod object and check the status against `expectedPodPhase`

Reply:
pod object whose status I need to check is passed as a parameter.